packages/conductor/e2e-test/incomingMessageHandler.e2e.test.ts

```diff
@@ -5,27 +5,22 @@ process.env.S3_AWS_SECRET_ACCESS_KEY = "test"
 import AuditLogApiClient from "@moj-bichard7/common/AuditLogApiClient/AuditLogApiClient"
 import createS3Config from "@moj-bichard7/common/s3/createS3Config"
 import putFileToS3 from "@moj-bichard7/common/s3/putFileToS3"
-import MockMailServer from "@moj-bichard7/common/test/MockMailServer"
 import waitForWorkflows from "@moj-bichard7/common/test/conductor/waitForWorkflows"
 import { type AuditLogApiRecordOutput } from "@moj-bichard7/common/types/AuditLogRecord"
 import AuditLogStatus from "@moj-bichard7/common/types/AuditLogStatus"
 import EventCode from "@moj-bichard7/common/types/EventCode"
-import { isError } from "@moj-bichard7/common/types/Result"
 import { randomUUID } from "crypto"
 import fs from "fs"
+import mailhogClient from "mailhog"
+
+const mailhog = mailhogClient()
 
 const INCOMING_BUCKET_NAME = "incoming-messages"
 const s3Config = createS3Config()
 
 describe("Incoming message handler", () => {
-  let mailServer: MockMailServer
-
-  beforeAll(() => {
-    mailServer = new MockMailServer(20002)
-  })
-
-  afterAll(() => {
-    mailServer.stop()
```

Comment:
Does MailHog have its own built in stop function?

Reply:
Good spot - this should be removed

Reply:
mailHog runs as a separate process and is integrated into the Docker environment now so we don't stop it as part of the tests